README.rst

```diff
@@ -392,3 +392,58 @@ Attach to DomU console with ``xu console``, use ``Ctrl-']'`` to exit console:
     thread_a: Hello World from cpu 0 on xenvm!
     thread_b: Hello World from cpu 0 on xenvm!
     thread_a: Hello World from cpu 0 on xenvm!
+
+Linux pv domain control
+=======================
+
+Create linux-pv-domu by using ``xu create`` with linux_pv_domu configuration and Xen
+domainId ``-d 3``.
```

Comment:
no need for "-d 3". I hope https://github.com/xen-troops/zephyr-xenlib/pull/89 will be merged it will be possible to see domid.

Reply:
not sure why we need this changes. Currently domid can be seen using xstat stat command. Setting predefined domain id saving me from the necessity of describing how can I get this id. So why not? This command is valid.